workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/androidx/WorkflowLifecycleOwner.kt

```diff
@@ -114,7 +113,7 @@ internal class RealWorkflowLifecycleOwner(
   /**
    * Weak reference ensures that we don't leak the view.
    */
```

Comment:
Need to remove/change the comment here (possibly elsewhere?).

What has changed that we are no longer concerned about the leak here? (Sorry if I should have already seen this pass by in a different discussion).

Reply:
And if it's still leaking, what we should really do is add some LeakCanary tests of our own out here to repro.

Reply:
> Nothing has changed

Well, this PR does add a lot of nulling out of the `View`, so that's a change.

Reply:
ack, yes OK, if we can repro with a test then SGTM!